framework/widget/src/main/java/org/apache/ofbiz/widget/model/ModelFormField.java

```diff
@@ -2799,6 +2811,26 @@ public String getValue(Map<String, Object> context) {
             return getModelFormField().getEntry(context);
         }
 
+        /**
+         * Gets disabled.
+         * @return the disabled
+         */
+        public FlexibleStringExpander getDisabled() {
```

Comment:
Can this be renamed `getDisabledSpec` to differentiate it from calling `getDisabled` with context arguments. (Similar to the change made to the `ModelFormField` class above.)

Does anything use this method? If not, we should remove it or make it private.

Reply:
right sorry I should have done the same here !